java/jsinterop/generator/closure/visitor/MemberCollector.java

```diff
@@ -118,13 +118,13 @@ protected void endVisitConstructor(FunctionType constructor) {
     // if the type is currently extended, copy the constructor on the extension point
     if (getJavaTypeRegistry().containsExtensionType(getCurrentJavaType())) {
       getJavaTypeRegistry()
-          .getExtensionType(getCurrentJavaType())
-          .addConstructor(Method.from(constructorMethod));
+              .getExtensionType(getCurrentJavaType())
+              .addConstructor(Method.from(constructorMethod));
     }
   }
 
   @Override
-  protected boolean visitField(StaticTypedSlot<JSType> jsField, boolean isStatic) {
```

Comment:
Looks like this needs to be put back?

Reply:
You mean the generic type? it does not compile with it any more.

Reply:
I think you might have an old version of closure-compiler in your bazel cache if that is what you are seeing - I haven't found a way for it to say "update your snapshots" yet. The best way is `bazel clean --expunge`, probably call this every time you get updates since you never know when it will require a change.

Then, the next build will correctly download a fresh copy, which should then allow this generic arg.

Reply:
> I haven't found a way for it to say "update your snapshots" yet. The best way is bazel clean --expunge, probably call this every time you get updates since you never know when it will require a change.

Fyi, this is because you're not supposed to use changing dependencies with Bazel, as you cannot get reproducible builds then (as exemplified here). With Bazel, you're supposed to use a specific SHA1 of the closure-compiler repository, and update it along with making the required changes in the code as the API changes. That way, any old commit will still build a year from now.
So, yes, `bazel clean --expunge`.

Reply:
i had this generic type before but when updated my fork it was removed, so it is not in the master branch of the original repo anymore

Reply:
i found the commit where those generic types removed 
https://github.com/google/jsinterop-generator/commit/746952f1741266362ca333db74537965e4fc88b3#diff-88a134d1688e01d441b4daa4146e03ee

Reply:
@tbroyer supposed to, yes, but at least so far in my limited exposure to bazel, this isn't consistently done, and "snapshot"-like behavior is encouraged/expected.

Reply:
@vegegoku I get it now, thanks - https://github.com/google/closure-compiler/commit/dda0e150c86cec64d3989102ad0d1ea26f0d21bc shows those generics being changed, which seems to mean all of these method args should be left as JSType, unless there is another commit on its way which removes them (which I assume there is).

That said, the change is already made for this file - you can remove these changes and rebase to latest master, and the change will be present in your local copy already